_tutorials/2016-05-22-getting-started-on-xrv64.md

```diff
@@ -0,0 +1,118 @@
+---
+published: false
+date: "2016-05-22 13:49 -0700"
+title: Getting Started on XRv64
+permalink: "https://xrdocs.github.io/application-hosting/tutorials/xrv64-quickstart"
+author: Lisa Roach
+excerpt: "Getting started with Cisco's XRv64 Vagrant Instance."
+tags: 
+  - vagrant
+  - iosxr
+  - cisco
+  - XRv64
+---
+### Quick Start Using XRv64 Box image
+Quick guide to get up and running with XRv64 and vagrant/Virtualbox.
+ 
+Install Virtualbox and vagrant tools to your mac:
+
+Source: http://sourabhbajaj.com/mac-setup/Vagrant/README.html
```

Comment:
See my other comment. This blog link seems to indicate homebrew as the primary way to install all the software components. I would recommend using the actual download links for the individual software pieces because blog links like this are transient and may be lost at any point.